### ISSUE TITLE
ABL过程未能成功初始化

### ISSUE THREAD

**Issue by mengshanfeng** [AbductiveLearning/ABL-HED#1]
在执行main.py的时候， CNN过程执行后，ABL模块未执行，跳出了帮助信息，如下：
** Now use autoencoder to pretrain the model **
Pretrain file exists, skip pretrain step.
** Initializing prolog **
swipl: Usage:
......

具体是src/main.py的168行： LL.init("-G10g -M6g")

**Comment by mengshanfeng** [comment]
@AbductiveLearning

**Comment by AbductiveLearning** [comment]
It seems that the problem lies in the environment for calling Prolog. Please paste all the error logs, which will be more helpful. Thanks!

**Comment by mengshanfeng** [comment]
Hi, there:
After executing the following cmd,
~/Downloads/ABL-HED/src$ python3 main.py
the following result occurs, which seems has nothing to do with the ABL process... 

Selected random seed is :  5105
Using TensorFlow backend.
/home/anaconda3/envs/abl/lib/python3.6/site-packages/h5py/__init__.py:34: FutureWarning: Conversion of the second argument of issubdtype from `float` to `np.floating` is deprecated. In future, it will be treated as `np.float64 == np.dtype(float).type`.
  from ._conv import register_converters as _register_converters

** Now use autoencoder to pretrain the model **
Pretrain file exists, skip pretrain step.
** Initializing prolog **
swipl: Usage:
    1) swipl [options] prolog-file ... [-- arg ...]
    2) swipl [options] [-o executable] -c prolog-file ...
    3) swipl --help     Display this message (also -h)
    4) swipl --version  Display version information
    4) swipl --arch     Display architecture
    6) swipl --dump-runtime-variables[=format]
                        Dump link info in sh(1) format
Options:
    -x state                 Start from state (must be first)
    -g goal                  Run goal (may be repeated)
    -t toplevel              Toplevel goal
    -f file                  User initialisation file
    -F file                  Site initialisation file
    -l file                  Script source file
    -s file                  Script source file
    -p alias=path            Define file search path 'alias'
    -O                       Optimised compilation
    --tty[=bool]             (Dis)allow tty control
    --signals[=bool]         Do (not) modify signal handling
    --threads[=bool]         Do (not) allow for threads
    --debug[=bool]           Do (not) generate debug info
    --quiet[=bool] (-q)      Do (not) suppress informational messages
    --traditional            Disable extensions of version 7
    --home=DIR               Use DIR as SWI-Prolog home
    --stack_limit=size[BKMG] Specify maximum size of Prolog stacks
    --table_space=size[BKMG] Specify maximum size of SLG tables
    --shared_table_space=size[BKMG] Maximum size of shared SLG tables
    --pce[=bool]             Make the xpce gui available
    --pldoc[=port]           Start PlDoc server [at port]

Boolean options may be written as --name=bool, --name, --no-name or --noname

**Comment by mengshanfeng** [comment]
@AbductiveLearning

**Comment by AbductiveLearning** [comment]
The problem is caused by the changes in SWI-Prolog's initialisation arguments. I will fix it soon, thanks for your feedback!
